sdk/communication/communication-identity/samples/v1/typescript/src/revokeTokens.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates how to use the CommunicationIdentityClient to
```

Comment:
Dom suggested that we rewrite these to sound more active and natural. So something like "Revokes a user's tokens"